electron/legendary/user.ts

```diff
@@ -20,19 +21,29 @@ export class LegendaryUser {
     return new Promise((res) => {
       const child = spawn(legendaryBin, command)
       child.stderr.on('data', (data) => {
-        console.log(`stderr: ${data}`)
+        logInfo(`stderr: ${data}`)
         if (`${data}`.includes('ERROR')) {
+          logError(`${data}`)
+          dialog.showErrorBox(
+            i18next.t('box.error.login.tittle', 'Cannot Login!'),
+            `${data}`
+          )
           return res('error')
         }
       })
       child.stdout.on('data', (data) => {
-        console.log(`stderr: ${data}`)
+        logInfo(`stderr: ${data}`)
         if (`${data}`.includes('ERROR')) {
+          logError(`${data}`)
+          dialog.showErrorBox(
+            i18next.t('box.error.login.tittle', 'Cannot Login!'),
+            `${data}`
+          )
           return res('error')
         }
       })
       child.on('close', () => {
-        console.log('finished login')
+        logInfo('finished login')
```

Comment:
You can also skip the prefix for now and i will take care of it in another pr :).

```suggestion
        logInfo('finished login', LogPrefix.Legendary)
```